appveyor.yml

```diff
@@ -1,15 +1,47 @@
-clone_folder: c:\xaxlib
 version: 2.16.{build}
-image: Visual Studio 2019
+image:
+  - Ubuntu
+  - Visual Studio 2019
+
```

Comment:
```suggestion
environment:
  matrix:
  - job_name: windows
  - job_name: linux
```

Reply:
Think you're rights. Bit unclear in the docs.

Reply:
AppVeyor don't like it